### ISSUE TITLE
no backbone ckpt referred.. && shape mismatch problems on custom dataset

### ISSUE THREAD

**Issue by qtw1998** [google/automl#365]
POST: IT'S  THE LATEST VERSION I'VE UPDATED
![image](https://user-images.githubusercontent.com/34357479/81216499-9232d180-900d-11ea-9599-086430a18754.png)

###########################shape mismatch##############################

When I load the ckpt of d1:
![image](https://user-images.githubusercontent.com/34357479/81216189-1d5f9780-900d-11ea-95cd-f9051fb39281.png)
although I set the yaml:
![image](https://user-images.githubusercontent.com/34357479/81216251-35371b80-900d-11ea-8940-fc96026581a9.png)

######################no backbone ckpt referred###########################
BESIDES `--backbone_ckpt=efficientnet-b1` I get `Failed to find any matching files for efficientnet-b1`

**Comment by fsx950223** [comment]
Remove the directory and unzip ckpt again.

**Comment by qtw1998** [comment]
> --num_classes=10

I've already set it in the yaml
![81216251-35371b80-900d-11ea-8940-fc96026581a9](https://user-images.githubusercontent.com/34357479/81252107-a0a5db00-9057-11ea-926e-222f0beb5f8a.png)
`CUDA_VISIBLE_DEVICES=3 python main.py --mode=train_and_eval --training_file_pattern=/cluster/home/qiaotianwei/bdd100k_data/train/*.tfrecord --validation_file_pattern=/cluster/home/qiaotianwei/bdd100k_data/val/*.tfrecord --val_json_file=/cluster/home/qiaotianwei/bdd100k_data/val/val.json --model_name=efficientdet-d1 --model_dir=save/ --train_batch_size=1 --eval_batch_size=4 --eval_samples=10000 --num_examples_per_epoch=70000 --num_epochs=300 --use_tpu=False --hparams=config.yaml --ckpt=efficientdet-d1`

**Comment by qtw1998** [comment]
> --num_classes=10

Actually It's  NO USE...

**Comment by fsx950223** [comment]
> > --num_classes=10
> 
> Actually It's NO USE...

I couldn't reproduce the problem.

**Comment by fsx950223** [comment]
In fact, you should debug such a problem by yourself.

**Comment by qtw1998** [comment]
> In fact, you should debug such a problem by yourself.

I thought it's a bug in the codes...

**Comment by fsx950223** [comment]
> > In fact, you should debug such a problem by yourself.
> 
> I thought it's a bug in the codes...

How do you prove it, could you find reason?

**Comment by qtw1998** [comment]
> > > In fact, you should debug such a problem by yourself.
> > 
> > 
> > I thought it's a bug in the codes...
> 
> How do you prove it, could you find reason?

So it's the reason why I ask for you

**Comment by fsx950223** [comment]
Closed.Due to I can't reproduce the problem.

**Comment by qtw1998** [comment]
> > > In fact, you should debug such a problem by yourself.
> > 
> > 
> > I thought it's a bug in the codes...
> 
> How do you prove it, could you find reason?

The bug is at FOCAL LOSS

**Comment by shreymohan** [comment]
I faced the same issue. Did you solve it?
